src/port/wiiu/WiiUImpl.cpp

```diff
@@ -59,9 +59,9 @@ void Init() {
     // make sure the required folders exist
     mkdir("/vol/external01/wiiu/", 0755);
     mkdir("/vol/external01/wiiu/apps/", 0755);
-    mkdir("/vol/external01/wiiu/apps/soh/", 0755);
+    mkdir(("/vol/external01/wiiu/apps/" + Window::GetInstance()->GetShortName()).c_str(), 0755);
```

Comment:
This will crash since at the time `Ship::WiiU::Init` is called, the Window instance hasn't been created yet.